fighthealthinsurance/common_view_logic.py

```
@@ -1812,7 +1813,14 @@ async def format_response(response: dict[str, str]) -> str:
             pubmed_context=pubmed_context,
             ml_citations_context=ml_citation_context,
         )
-        filtered_appeals: Iterator[str] = filter(lambda x: x != None, appeals)
+        # Only filter for professional tone if professional_to_finish is True
+        # This ensures only provider-voice appeals are included when requested
+        if professional_to_finish:
+            filtered_appeals: Iterator[str] = filter(
```

Ruff (0.8.2)

`Iterator` may be undefined, or defined from star imports

(F405)